### PR TITLE
feat: adding german translation

### PR DESCRIPTION
**Pull request by RichardMEN11**
I have added the translation for the german language for this package. If there is something to improve feel free to let me know. Thanks a lot!

### CODE REVIEW

File: vuepress/guide/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Per default the locale is set to english `en`.
 
 Per default the word per minute is set to `300`.
 
-At the moment there is only 5 supported locales: `en`, `fr`, `es`, `cn` and `ja`. If a
+At the moment there is only 5 supported locales: `en`, `fr`, `es`, `cn`, `ja` and `de`. If a
```
Comment (lbenie) [inline vuepress/guide/README.md:36]: Thanks for contributing!

```suggestion
At the moment there is only 6 supported locales: `en`, `fr`, `es`, `cn`, `ja` and `de`. If a
```

**Comment by lbenie** [comment]
@all-contributors please add @RichardMEN11  for code, docs

**Comment by lbenie** [comment]
Would you mind adding the fix for the docs in the `fr` folder as well 🙏🏽 🙇🏽

File: vuepress/guide/README.md (hunk 1 above)
Comment (RichardMEN11) [reply vuepress/guide/README.md]: Thanks. Its done! :-)

**Comment by RichardMEN11** [comment]
> Would you mind adding the fix for the docs in the `fr` folder as well 🙏🏽 🙇🏽

This should be also done.

File: vuepress/fr/guide/README.md [hunk 2]
```diff
@@ -28,13 +28,13 @@ ou Yarn
 L'api est assez simple. Voici la définition des types pour ce module.
 
 
-Il s'agit d'une fonction simple qui prend les données comme argument requis avec les mots par minute et les paramètres régionaux comme arguments facultatifs.
+Il s'agit d'une fonction simple qui prend les données comme argument requis avec les mots par minute et les paramètres régionaux comme arguments facultatifs.Ì
```
Comment (lbenie) [inline vuepress/fr/guide/README.md:31]:
```suggestion
Il s'agit d'une fonction simple qui prend les données comme argument requis avec les mots par minute et les paramètres régionaux comme arguments facultatifs.
```

**Pull request merged**